src/Dev/SapphireTest.php

```diff
@@ -1793,7 +1793,7 @@ public static function tearDownAfterClass()
 
     /**
      * @return FixtureFactory|false
-     * @deprecated 4.0.0:5.0.0
+     * @deprecated 4.0.0
```

Comment:
The version in the below notice needs updating, and the message needs to be standardised
And add the message here.

Reply:
Updated